src/config-schema.coffee

```diff
@@ -100,6 +100,10 @@ module.exports =
         description: 'Automatically open an empty editor on startup.'
         type: 'boolean'
         default: true
+      enableAutoUpdate:
```

Comment:
What do you think about just making this `automaticallyUpdate` instead of `enableAutoUpdate`.

Looks like most of (or all) the other config items don't start with `enable`, such as `scrollPastEnd` instead of `enableScrollPastEnd`.